dependencies.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import sys
+import subprocess
+import json
+
+# Set default logging level to INFO
+logging.basicConfig(level="INFO")
+
+# Check if current python version is 3.8
+if sys.version_info < (3, 8):
+    logging.fatal('McPy needs Python version 3.8.0 or higher to run! Current version is %s.%s.%s' % (
+        sys.version_info.major, sys.version_info.minor, sys.version_info.micro))
+    sys.exit(-4)
+
+# If the OS is Linux, the module "distro" will import
+try:
+    import distro
+except ModuleNotFoundError: # Windows and MacOS will not find this module, because they don't have distributions
+    pass
+
+# Make sure pip is installed
+logging.info("Making sure pip is installed...")
+try: # Debian and Ubuntu require pip to be installed differently because they disable ensurepip
+    linuxDistro = distro.linux_distribution()
+    if linuxDistro[0] == "Debian" or linuxDistro[0] == "Ubuntu":
+        try:
+            subprocess.check_call(['sudo', 'apt', 'install', 'python3-pip'])
```

Comment:
Is there a way to check if pip is installed ?
Because if it is, you still run apt install.
(does `import pip` work to check if it's installed ?)

Reply:
There is a way with a module that comes installed with python. It is ensurepip, but the issue is that Debian-based distros disable this module for some reason